draft-ietf-quic-transport.md

```diff
@@ -3461,9 +3468,14 @@ ACK frames in response to other packets.  Implementations MUST NOT send more
 than one packet containing only an ACK frame per received packet that contains
 frames other than ACK and PADDING frames.  Packets containing frames besides
 ACK and PADDING MUST be acknowledged immediately or when a delayed ack timer
-expires. The delayed ack timer MUST NOT delay an ACK for longer than an RTT or
-the alarm granularity.  This ensures an ACK frame is sent at least once per RTT
-if new packets needing acknowledgement were received.
+expires.
+
+The receiver's delayed ack timer MUST NOT delay an ACK for longer than an RTT
+or the value indicated by the `max_ack_delay` transport parameter.
```

Comment:
you say that it might exceed `max_ack_delay` in the recovery PR, so which is it?